src/app.rs

```diff
@@ -797,7 +954,8 @@ impl eframe::App for RustyAutoClickerApp {
                             .clicked()
                         {
                             // Start autoclick, first click is delayed
-                            Self::start_autoclick(self, 0u64);
+                            // Self::start_autoclick(self, 0u64);
```

Comment:
We'll have to figure out a way to reimplement this, see comment on L369; maybe we can skip `is_executing` and go back to `is_autoclicking` since the humanlike movement only really affects 1 of the application modes.

Reply:
Uncomment this line please to re-enable it